fast64_internal/oot/ActorList.xml

```diff
@@ -0,0 +1,3266 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Table KeyType="System.UInt16" ValueType="System.String">
+  <Actor Category="ACTORCAT_PLAYER" ID="ACTOR_PLAYER" Key="0000" Name="Link / Spawn point - PLAYER">
+    <Parameter Params="0000"> Invisible, Can't Control //Used for cutscenes without Link</Parameter>
+    <Parameter Params="0100"> Master Sword Pull and Drop Animation //Overwrites coordinates</Parameter>
+    <Parameter Params="0200"> Spawn: Blue Warp //used when returning from a blue warp</Parameter>
+    <Parameter Params="0300"> Spawn: Immobile</Parameter>
+    <Parameter Params="0400"> Spawn: Exit via Grotto //Y velocity = 12</Parameter>
+    <Parameter Params="0500"> Spawn: Warp Song</Parameter>
+    <Parameter Params="0600"> Spawn: Farore's Wind</Parameter>
+    <Parameter Params="0700"> Spawn: Thrown Out</Parameter>
+    <Parameter Params="0D00"> Stand</Parameter>
+    <Parameter Params="0E00"> Slow Walk Forward</Parameter>
+    <Parameter Params="0F00"> Retain Previous Velocity //Walk speed set by last exit</Parameter>
+    <Property Mask="0x0000" Name="None"/>
+    <Property Index="1" Mask="0x00FF" Name="Camera ID (nullable)"/>
+    <Notes>Last 2 digits = Camera Initial Focus //Data defined by scene Link spawns in
++00FF No Special Positioning</Notes>
+  </Actor>
+  <Actor Category="ACTORCAT_ENNEMY" ID="ACTOR_EN_TEST" Key="0002" Name="Stalfos - EN_TEST" ObjectID="OBJECT_SK2">
```

Comment:
```suggestion
  <Actor Category="ACTORCAT_ENEMY" ID="ACTOR_EN_TEST" Key="0002" Name="Stalfos - EN_TEST" ObjectID="OBJECT_SK2">
```

Reply:
Apparently I got confused with how you write it in french, gonna fix that